app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,43 @@
-<h2>Log in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+<div class ="container">
+  <div class="user-sessions">
+    <div class="card-signin" >
+      <div class="login-heading" style="font-weight: lighter;"> 
+      <%= image_tag("/login-icon.png", :alt => "img") %>
+       Login
+      </div>
+      <div class="card-body">
+        <div class="parsley-form">
+          <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    
+            <div class="form-group">
+                <%= f.label :login , "Username or Email" ,class:'required no-transform label' %><br/> 
+                <%= f.text_field :login,class:'form-control ', autofocus: true %>
+                <label style="color: #DCDCDC"><i>Hint: username= firstname lastname<i></label>
+            </div>
+        
+            <div class=" form-group" >
+              <%= f.label :password ,class:"required no-transform label"%><br />
+              <%= f.password_field :password, class:'form-control' ,autocomplete: "current-password" %>
+            </div>
+            <div class="bottom-grid">
 
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
+              <% if devise_mapping.rememberable? %>
+                <div class=" form-group form-check" >
+                  <%= f.check_box :remember_me %>
+                  <%= f.label :remember_me %>
+                </div>
+              <% end %>
+                <%= f.submit "Log in" , class:"btn btn-primary"%>
+                <%= link_to "Lost your password?", new_password_path(User),class:'forget-link' %></div> 
```

Comment:
Spacing issue
Also why not `new_password_path(resource_name)` instead of `new_password_path(User)`